Ladybird/WebContent/CMakeLists.txt

```diff
@@ -34,6 +34,14 @@ endif()
 target_include_directories(webcontent PUBLIC ${LADYBIRD_SOURCE_DIR}/Userland/Services/)
 target_include_directories(webcontent PUBLIC ${LADYBIRD_SOURCE_DIR}/Userland/)
 target_include_directories(webcontent PUBLIC ${CMAKE_CURRENT_BINARY_DIR}/..)
+
+if (ANDROID)
+    # FIXME: Not super clear why this is needed for Android.
+    # Without it, we get errors about webcontent having include directories prefixed in its source directory.
+    # See https://stackoverflow.com/q/25676277/2178519
+    set_target_properties(webcontent PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "")
```

Comment:
For this, we want any build-dir directories to have `$<BUILD_INTERFACE:${SOME_DIRECTORY}>`  That might be due to transitive dependencies from third party libraries.

Reply:
We can ignore `$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/whatever>`, because we don't particularly care about downstream applications trying to use these at the moment.

Reply:
That makes sense, I've updated it to mark the include directories as BUILD_INTERFACE.